### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by aviggiano**
- Create ElasticSearch service
- Attach to Cognito (TF resources TBD)
- Create stream of Cloudwatch Logs to ElasticSearch
- Create automatic subscription of Cloudwatch Logs

NB: untested

### CODE REVIEW

File: lambda/cloudwatch.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export async function isSubscribed (logGroupName: string): Promise<boolean> {
+  const response = await logs.describeSubscriptionFilters({ logGroupName }).promise()
+  return response.subscriptionFilters ? response.subscriptionFilters.length > 0 : false
+}
+
+export async function putSubscription (logGroupName: string, destinationArn: string):
+Promise<unknown> {
+  const params = {
+    filterName: `${logGroupName}-filter`,
```
Comment (aviggiano) [inline lambda/cloudwatch.ts]: Why is this necessary? Can I use simply `logGroupName`?

Comment (papukweh) [reply lambda/cloudwatch.ts]: Yes, you can use `logGroupName`. It's just a descriptive name for the filter.

File: policies/lambda_policy.tpl [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "Statement": [
+    {
+      "Sid": "CopiedFromTemplateAWSLambdaVPCAccessExecutionRole1",
+      "Effect": "Allow",
+      "Action": [
+        "logs:Create*",
+        "logs:Describe*",
+        "ecs:listTasks",
+        "es:ESHttpPost",
+        "sqs:SendMessage"
```
Comment (papukweh) [inline policies/lambda_policy.tpl]: I think these `ecs` and `sqs` policies are not necessary

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  region = "us-east-1"
+}
```
Comment (papukweh) [inline main.tf]: I think a module shouldn't have a `provider` block, but if it's necessary, maybe we should let the `region` argument be an input variable?
